roles/ora-host/tasks/main.yml

```diff
@@ -433,3 +431,28 @@
     - "{{ listdisks }}"
   when: asm_disk_management == "asmlib"
   tags: asm-disks
+  
+  - name: (asmlib) ASM listing results
+  debug:
+    msg:
+    - "{{ item.stderr_lines }}"
+    - "{{ item.stdout_lines }}"
+  with_items:
+    - "{{ asmconfig }}"
+    - "{{ listdisks }}"
+  when: asm_disk_management == "asmlib"
+  tags: asm-disks
+
+- name: Block http to metadata 
+  iptables:
+    chain: OUTPUT
+    protocol: tcp
+    destination: 169.254.169.254
+    destination_port: "80"
+    jump: DROP
+
+- name: config iptables service
+  command: chkconfig iptables on
+
+- name: persist iptables rules
```

Comment:
This change may break more modern firewall configurations (say with firewalld), as it clears out existing tables completely.  https://serverfault.com/questions/674874/is-there-a-way-to-run-just-save-with-firewalld-in-rhel7 has some ideas with firewall-cmd that might help.

Reply:
I agree, I was able to do the same functionality with firewalld but this would take us back to the comment about enabling or disabling firewalld, I would suggest that we give the user the option to choose, if he wants firewalld enabled then we block metadata server with firewalld (and we would need to enable interconnect through zone configuration), if he wants it disabled we use iptables.